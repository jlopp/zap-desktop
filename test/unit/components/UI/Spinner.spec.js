import React from 'react'
import Spinner from 'components/UI/Spinner'
import renderer from 'react-test-renderer'

describe('component.UI.Spinner', () => {
  it('should render correctly', () => {
    const tree = renderer.create(<Spinner />).toJSON()
    expect(tree).toMatchSnapshot()
  })
})
