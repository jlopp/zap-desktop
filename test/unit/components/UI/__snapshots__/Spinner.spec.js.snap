// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`component.UI.Spinner should render correctly 1`] = `
.c0 {
  width: 1em;
  height: 1em;
  border-radius: 999px;
  border: 1px solid rgba(235,184,100,0.1);
  border-left-color: rgba(235,184,100,0.6);
  display: inline-block;
  -webkit-animation: iVXCSc 1s linear infinite;
  animation: iVXCSc 1s linear infinite;
}

<div
  className="c0"
  width="1em"
/>
`;
